docs/src/main/asciidoc/cli-tooling.adoc

```diff
@@ -43,24 +53,56 @@ If you have built Quarkus locally, you can use that version:
 jbang app install --force --name qss ~/.m2/repository/io/quarkus/quarkus-cli/999-SNAPSHOT/quarkus-cli-999-SNAPSHOT-runner.jar
 ----
 
-Once installed `quarkus` will be in your PATH and if you run `quarkus --version` it will print the installed version:
+Once installed `quarkus` will be in your `$PATH` and if you run `quarkus --version` it will print the installed version:
 
-[source,shell]
+[source,shell,subs=attributes+]
 ----
 quarkus --version
-Client Version {quarkus-version}
+{quarkus-version}
 ----
 
 [CAUTION]
 .Use a recent JBang version
 ====
 If you get an error about `app` not being readable then you probably
-have a `jbang` version older than v0.56.0 installed. Please remove or upgrade it to a recent version.
+have a JBang version older than v0.56.0 installed. Please remove or upgrade it to a recent version.
+
+If you are installing JBang for the first time, start a new session to update your `PATH`.
+====
```

Comment:
Should we add / update / replace this note with one that talks about marking the content as trusted?

Reply:
Yeah I thought about removing it but decided to keep it for now and let Max update this part when he sees fit.